### DISCUSSION TITLE
Non-Ascii is fucked

### DISCUSSION THREAD

**Discussion by SeanMatts** [lptstr/winfetch#183] [Q&A]
![image](https://user-images.githubusercontent.com/103733984/225715650-342b3a73-2544-47f5-9c2d-b1cc5643c10f.png)
The Image Kinda says it all

**Comment by rashil2000** [comment]
What?

**Comment by SeanMatts** [comment]
I want to use a PNG as my image, and use it as non-ASCII, but all the black pixels are supposed to be transparent.

**Comment by rashil2000** [comment]
Link to image?
